config/Data_Schema.toml

```diff
@@ -0,0 +1,469 @@
+[snapshot_id]
+Description = "nan"
+Deduced_Data_Type = "nan"
+Nullable = "nan"
+Current_Data_Type = "str"
+Length = "nan"
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = ["nan"]
+
+[reference]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = 11001603625.0
+Max_values = 19891309165.0
+Possible_Categorical_Values = ["nan"]
+
+[period]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = 6
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = [202012]
+
+[survey]
+Description = "All values are 002"
+Deduced_Data_Type = "str"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = ["002"]
+
+[formid]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "int"
+Length = 2
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = [20, 21]
+
+[status]
+Description = "nan"
+Deduced_Data_Type = "str"
+Nullable = 0.0
+Current_Data_Type = "str"
+Length = ">=1"
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = ["Clear", "Clear - overridden", "Form saved", "Clear - overridden SE", "Form sent out", "Check needed", "Combined child (NIL2)", "Out of scope (NIL3)", "Ceased trading (NIL4)", "Dormant (NIL5)", "Part year return (NIL8)", "No UK activity (NIL9)"]
+
+[statusencoded]
+Description = "nan"
+Deduced_Data_Type = "int"
+Nullable = 0.0
+Current_Data_Type = "int"
+Length = 3
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = [100, 200, 201, 210, 211, 301, 302, 303, 304, 305, 308, 309]
+
+[receiptdate]
+Description = "Datetime format = %Y-%m-%d %H:%M:%S.%f+00"
+Deduced_Data_Type = "Datetime"
+Nullable = 0.0
+Current_Data_Type = ["None","str"]
+Length = "nan"
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = ["nan"]
+
+[lockedby]
+Description = "All empty strings"
+Deduced_Data_Type = "?"
+Nullable = 1.0
+Current_Data_Type = ["None","str"]
+Length = 0
+Min_values = "nan"
+Max_values = "nan"
+Possible_Categorical_Values = ["nan"]
+
+[lockeddate]
+Description = "All None type"
+Deduced_Data_Type = "?"
```

Comment:
pandas.NA maybe?

Reply:
Yeah, I think so. Certainly, until further data/something else comes through to suggest otherwise!